src/main/groovy/io/seqera/wave/service/scan/DockerScanStrategy.groovy

```diff
@@ -100,6 +100,9 @@ class DockerScanStrategy extends ScanStrategy {
             }
         }
 
+        wrapper.add('--entrypoint')
+        wrapper.add('/usr/local/bin/scan.sh')
+
```

Comment:
it should not go as entry point, but first element in the command composed by `buildScanCommand`